CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### 🐞 Fixed
 - Fix an edge case with a semicolon in user name [#3275](https://github.com/GetStream/stream-chat-swift/pull/3275)
 
+### 🔄 Changed
```

Comment:
Isn't this a fix? 🤔

Reply:
From SwiftUI's point of view it is a fix. I'll change it around.

Reply:
Not really, more like a change we're doing. It was never a bug.

Reply:
The context here is LLC - and we change the equatable conformance. The edit message bug will be put in the SwiftUI Fixed changelog accordingly.

Reply:
Makes sense, I'm fine with either Change or a fix in this case 👍